smash/fcore/operator/md_routing_operator.f90

```diff
@@ -257,18 +334,24 @@ subroutine lr_time_step(setup, mesh, options, ac_qtz, ac_llr, ac_hlr, ac_qz)
 
     end subroutine lr_time_step
 
-    subroutine kw_time_step(setup, mesh, options, ac_qtz, ac_akw, ac_bkw, ac_qz)
+    subroutine kw_time_step(setup, mesh, options, returns, t, ac_qtz, ac_akw, ac_bkw, ac_qz)
+        ! Bug fixed on the parallel adjoint linked to variable names by changing of variable names :
+        ! Added local variable t instead of time_step as input to kw_time_step -> no change
+        ! Added local variable t_returns instead of time_step_returns to kw_time_step
+        ! -> modified openMP, unmodified non-openMP
```

Comment:
Je comprends pas bien le commentaire.
- ``Added local variable t instead of time_step as input to kw_time_step -> no change``. Moi ca ne marchait pas avec ``time_step``
- ``Added local variable t_returns instead of time_step_returns to kw_time_step  -> modified openMP, unmodified non-openMP``. Ca veut dire que ca a modifié l'adjoint avec OpenMP mais pas celui sans ?